packages/config-cli/bmc.js

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env node
```

Comment:
this file should be referenced by the bin field in the package.json, it also wont be published.

Its best to put this into `./bin/bmc.mjs` (being specific is super helpful for these)

then add `bin/` to the files exported and add a `bin` section to map it to `basemaps-screenshot` ?